### PR TITLE
BSTOR-19 Set background-color to brand's color when product is a t-shirt

### PR DESCRIPTION
**Pull request by Pacane**
image
image

### CODE REVIEW

**Comment by meriouma** [comment]
Not sure what are the next steps and what is styled depending on the brand, but I think adding a class to the main panel or body might be better, otherwise LGTM

**Comment by meriouma** [comment]
Nevermind, I thought the color was depending on the brand, but it's for the shirt only?

**Comment by Pacane** [comment]
We already do that, but in this case we need this only when it's a t-shirt... it's kind of an edge case. Maybe we should add another class when it's a t-shirt or something. I'll check with @LarryMatte

**Comment by Pacane** [comment]
@meriouma yeah it's only for t-shirts... because design lol

edit: In fact it's brand dependant, but also "t-shirt dependant"...

File: client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java [hunk 1]
```diff
@@ -119,14 +129,23 @@ public void showSharePanel() {
 
     @Override
     public void setProduct(ProductDto productDto) {
-        $(brandName).text(productDto.getBrand().getValue());
+        Brand brand = productDto.getBrand();
```
Comment (jclariviere) [inline client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java:134]: Group `Brand brand = ...` and `Product product = ...` together.

**Comment by jclariviere** [comment]
LGTM

**Pull request merged**